helm-chart/templates/rbac.yaml

```diff
@@ -56,12 +56,14 @@ rules:
   verbs:
   - list
   - watch
+{{- if .Values.controller.reportIngressStatus.enable }}
```

Comment:
we need `get` as well for Ingress resources

Reply:
thought i added that weeks ago